packages/plugin-ext/src/common/plugin-api-rpc-model.ts

```diff
@@ -434,6 +434,12 @@ export interface WorkspaceFolder {
     index: number;
 }
 
+export interface IOpenFolderAPICommandOptions {
+    forceNewWindow?: boolean;
+    forceReuseWindow?: boolean;
+    noRecentEntry?: boolean;
+}
+
```

Comment:
@akosyakov is this is the right place for this interface?

Reply:
I think we can keep it in the vscode command contribution file for now, it is not used anywhere else

Reply:
Done